docs/guides/ubuntu/_include/live-environment.rst

```diff
@@ -0,0 +1,25 @@
+Configure Live Environment
+--------------------------
+
+Open a root shell
+~~~~~~~~~~~~~~~~~~~~~~
+
+During boot of the installation media, you'll be greeted with the installer, giving you the option to `Try ubuntu` or
```

Comment:
This should be removed; people reading the guide should be versed enough with Linux/Live CDs to know that they're not using the official installer and instead need to get to a terminal.

Reply:
Done